### PR TITLE
cnf-tests: Update README.md

### PR DESCRIPTION
**Pull request by liornoy**
This commit updates the CNF-tests README to reflect the changes introduced in the CNF-tests split modification.

All the previous details about the latency tests have been removed, and a reference to the official documentation has been provided.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,10 +78,16 @@ RPMS_NODE_ROLE is optional and defaults to `node-role.kubernetes.io/worker`.
...
 
-A dockerized version of CNF tests is available at [quay.io/openshift-kni/cnf-tests](https://quay.io/openshift-kni/cnf-tests).
-For more details on how to use it, please check the [corresponding docs](cnf-tests/README.md).
+A dockerized version of the Node Tuning Operator latency test suite available at [quay.io/openshift-kni/cnf-tests](https://quay.io/openshift-kni/cnf-tests).
+For more details on how to use it, please check the [official docs](https://docs.openshift.com/container-platform/4.14/scalability_and_performance/cnf-performing-platform-verification-latency-tests.html).
+
+### The Conformance tests
+
+The conformance test suites verify that the operators we maintain are working properly on CNF enabled cluster.
+They are also used to run the nightly upstream ci jobs, triggered from the openshift/release ci-operator [link](https://github.com/openshift/release/blob/master/ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh).
```
[the remark this answers is not in the document]
Comment (liornoy) [reply README.md]: fixed, thanks.

File: cnf-tests/README.md [hunk 2]
```diff
@@ -1,550 +1,38 @@
...
+  * [Test Failure Report](#test-failure-report)
+* [Gatekeeper](#gatekeeper)
 
-# The CNF Tests image
+    
+# The Conformance Tests 
 
-The [CNF tests image](https://quay.io/openshift-kni/cnf-tests) is a containerized version of the CNF conformance test suite.
-It's intended to be run against a cluster where all the components required for running CNF workloads are installed.
+The conformance test suites verify that the operators we maintain are working properly on CNF-enabled clusters. The suites are "validationsuite", "configsuite", and "cnftests". The underlying Ginkgo test suites run by each suite can be found in the `TESTS_PATHS` variable in `hack/common.sh`.
```
[the remark this answers is not in the document]
Comment (liornoy) [reply cnf-tests/README.md]: added the clarification

File: cnf-tests/README.md [hunk 3]
```diff
@@ -1,550 +1,38 @@
...
-# The CNF Tests image
+    
+# The Conformance Tests 
 
-The [CNF tests image](https://quay.io/openshift-kni/cnf-tests) is a containerized version of the CNF conformance test suite.
-It's intended to be run against a cluster where all the components required for running CNF workloads are installed.
+The conformance test suites verify that the operators we maintain are working properly on CNF-enabled clusters. The suites are "validationsuite", "configsuite", and "cnftests". The underlying Ginkgo test suites run by each suite can be found in the `TESTS_PATHS` variable in `hack/common.sh`.  
 
-This include:
+To prevent dependency conflicts, we use git submodules to store external repositories: `cluster-node-tuning-operator`, `metallb-operator`, and `sriov-network-operator`, in the path `cnf-tests/submodules`.
```
[the remark this answers is not in the document]
Comment (liornoy) [reply cnf-tests/README.md:18]: yes this is important. added, thanks.

File: cnf-tests/README.md [hunk 4]
```diff
@@ -1,550 +1,38 @@
...
-### Cleaning Up
-
-After running the test suite, all the dangling resources are cleaned up.
+**KUBECONFIG**: The path to the kubeconfig for the cluster.  
+**SKIP_TESTS**: Translates to the ginkgo skip flag.  
+**FEATURES** and **FOCUS_TESTS**: Translates to the ginkgo focus flag.  
+**GINKGO_PARAMS**: Provide any additional valid ginkgo parameters.  
+**EXTERNAL_SUITES**: Specifies which external test suites to run; options include: (integration/metallb/nto/sriov/nto-performance)  
+**TESTS_REPORTS_PATH**: Path where the test reports will be stored.
+**FAIL_FAST**: Set to "true" to enable the ginkgo `--fail-fast` flag.
```
[the remark this answers is not in the document]
Comment (liornoy) [reply cnf-tests/README.md]: added

**Review by karampok — APPROVED** [review]
/lgtm

**Comment by zeeke** [comment]
/override ci/prow/e2e-aws-ci-tests

Failure is not related to these changes

**Pull request merged**